builtin/game/chatcommands.lua

```diff
@@ -679,19 +679,36 @@ core.register_chatcommand("status", {
 })
 
 core.register_chatcommand("time", {
-	params = "<0...24000>",
+	params = "<0..23>:<0..59> | <0..24000>",
 	description = "set time of day",
 	privs = {settime=true},
```

Comment:
The priv check should be done inside the method, and only be done when the person wants to get the time, to make it useful on servers where not every player has settime priv.